### ISSUE TITLE
Investigate and fix "null is not of subtype string" crash seen in processIncomingEvents

### ISSUE THREAD

**Issue by vishalxl** [vishalxl/nostr_console#70]
To attach pic later

```
Unhandled exception:
type 'Null' is not a subtype of type 'String'
#0      Store.handleChannelEvents (package:nostr_console/tree_ds.dart:815)                                #1      Store.processIncomingEvent.<anonymous closure> (package:nostr_console/tree_ds.dart:1508)          #2      SetMixin.forEach (dart:collection/set.dart:142)                                                   #3      Store.processIncomingEvent (package:nostr_console/tree_ds.dart:1445)
#4      processAnyIncomingEvents.<anonymous closure> (package:nostr_console/console_ui.dart:27)
#5      new Future.delayed.<anonymous closure> (dart:async/future.dart:424)
#6      Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18)
#7      _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398)
#8      _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429)
#9      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192)

```

**Comment by vishalxl** [comment]
This was fixed in a build on 4 jan 2023.

**Comment by vishalxl** [comment]
Was fixed in commit [here](https://github.com/vishalxl/nostr_console/commit/c50607119e38e5ffc53840423cb6947324eb561d)

Was included in release 0.3.5-beta.

**Comment by vishalxl** [comment]
[release with fix is 0.3.5-beta](https://github.com/vishalxl/nostr_console/releases/tag/v0.3.5-beta)
